test_parse.py

```
@@ -0,0 +1,18 @@
+import ast
+try:
+    with open("setup/launch.py", "r") as f:
+        content = f.read()
+    ast.parse(content)
+    print("Parsed launch.py successfully")
+except Exception as e:
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)